### PR TITLE
docker-test-pr

### PR DESCRIPTION
**Pull request by hyongtao**
Author: Yongtao Huang <hyongtao@vmware.com>
Date:   Wed Jun 23 19:44:08 2021 +0800

    The dockerfiles are apply to deploy and test TVM environments (including CPU and NVIDIA_GPU) easily.
    Long log:
    - There are some errors when we re-build official dockerfile like demo_cpu: https://discuss.tvm.apache.org/t/error-about-tvm-docker/10301
    - I have build CPU_TVM and NVIDIA_GPU_TVM docker, which includes inference envs and code.
    - This can help us verify and deploy our ML model directly.
    Signed-off-by: Yongtao Huang <hyongtao@vmware.com>

### CODE REVIEW

**Comment by comaniac** [comment]
@areusch @hogepodge could you help review this PR as you guys are working on docker related features recently?

**Comment by hogepodge** [comment]
@comaniac @hyongtao I'm on PTO until Monday, but will review the builds once I'm back at the office on Monday. I made some line edits to the Docker files. I would like to see more documentation on how to use the added test files. I'm also curious to know why they are docker specific tests and not more general integration tests.

File: docker/test/Dockerfile.CPU [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# 2. Install TVM
+WORKDIR /workspace/
+RUN git clone --recursive https://github.com/apache/tvm tvm
+WORKDIR /workspace/tvm/
+RUN mkdir build && cp cmake/config.cmake build
+RUN cd build && echo set\(USE_LLVM llvm-config-9\) >> config.cmake && cmake .. && make -j4
+
+# 3. Prepare Python sitpackage
+RUN apt-get install -y python3-pip && pip3 install --upgrade pip
+RUN pip3 install numpy scipy decorator attrs tornado psutil tensorflow pytest
```
Comment (areusch) [inline docker/test/Dockerfile.CPU:28]: i'm not sure things are going to work super well if you just install latest

**Review by areusch — COMMENTED** [review]
hi @hyongtao, thanks for your contribution. i'd like to better understand your use case for these tests. it seems like it may be specific to your model deployment rather than something generally reusable across TVM/all TVM models. could you give me some more context on how you use this? we currently do have integration tests on a limited number of models, but cannot afford to add integration tests for all models to the TVM CI (it is already quite overloaded).

**Comment by masahi** [comment]
@hyongtao Can you update?

**Comment by jroesch** [comment]
This PR appears to be out of date, please feel free to reopen it if this is not the case.

As part of the new year we are attempting to triage the project's open pull requests to ensure that code which
is ready for review and/or merging receives adequate attention.

Thanks again for your contribution, and feel free to reach out to discuss these changes.

**Pull request closed without merging**